src/chocolatey/infrastructure.app/configuration/ConfigurationOptions.cs

```diff
@@ -120,6 +123,12 @@ public static OptionSet OptionSet
 
             if (configuration.HelpRequested)
             {
+                if (configuration.ShowOnlineHelp)
+                {
+                    System.Diagnostics.Process.Start(string.Format("https://docs.chocolatey.org/en-us/choco/commands/{0}", configuration.CommandName));

```

Comment:
Not 100% sure, but if I remember correctly, this won't work on all/most Linux distros (possibly not OSX as well).

It may be just for older variants, though. @TheCakeIsNaOH are you able to confirm or deny whether just starting a URL works on the distros that you are aware of?

Reply:
I just tested this on a Debian system with a GUI, and it is working for me, but it did not work on a headless system.